### PR TITLE
Updated e2e test docker image

### PR DESCRIPTION
**Pull request by BenjaminKeller**
_**Disclaimer:**_ This branch contains the old commit fb13aa0cefa2dd4d0a4f8bc434d013015f8d0269 from the old branch check_for_eidclient. I split it into several new commits.

The e2e tests from the forked repository contained an old npm package called `jasmine-node`. I moved it to the npm package `jasmine` which is more close to the normal jasmine. Additionally I updated the frisby library from 0.8 to 2.0.2 so we can use new features.

### Known Issue
For Windows 7 I had to adapt the `docker-compose.test.yml`. Otherwise the host name couldn't be resolved. The `extra_hosts` option adds the line to the container's hosts file.
```
version: '2.1'

# mostly extending from main .yml
services:
  test-e2e:
    build: test/e2e
    restart: "no"
    environment:
    - BOILERPLATE_DOMAIN
    #- BOILERPLATE_IPV4_16PREFIX
    #- BOILERPLATE_IPV6_ADDRESS
    depends_on:
    - www
    networks:
    - front
    extra_hosts:
    - "eid.com:192.168.99.100"
```

### CODE REVIEW

File: test/e2e/package.json [hunk 1]
```diff
@@ -1,15 +1,17 @@
 {
   "name": "test-e2e",
   "version": "1.0.0",
-  "description": "End-to-end tests for the Django REST Boilerplate application",
+  "description": "End-to-end tests for the EID project",
```
Comment (nils-wisiol) [inline test/e2e/package.json]: Please decide for a name and update accordingly. Plain "EID" is misleading, as this repository shall provide an EID based authentication service.

**Review by nils-wisiol — CHANGES_REQUESTED** [review]
Other than the name, this lgtm.

Also, please update the repository to run CI for PRs.

File: test/e2e/package.json (hunk 1 above)
Comment (BenjaminKeller) [reply test/e2e/package.json]: I opened a new issue #35 and will change the name after a decision.

**Comment by BenjaminKeller** [comment]
Updated project name.

**Pull request merged**